packages/valory/agents/hello_world/aea-config.yaml

```diff
@@ -52,10 +52,21 @@ dependencies:
   open-aea-ledger-ethereum: {}
 default_connection: null
 ---
+public_id: valory/hello_world_abci:0.1.0
+type: skill
+models:
+  params:
+    args:
+      tendermint_url: ${TENDERMINT_URL:str:http://localhost:26657}
+      tendermint_com_url: ${TENDERMINT_COM_URL:str:http://localhost:8080}
```

Comment:
Same

Reply:
why defined on the agent and not on the service?

Reply:
So these are environment overrides which mean these variables will be picked from the env at the runtime. These env vars will be defined by the service and used by the agent that's why they're defined on agent config